### PR TITLE
multiple improvements to devcontainer

### PR DESCRIPTION
**Pull request by jonas-kaufmann**

### CODE REVIEW

**Review by q713 — APPROVED** [review]
LGTM

File: .devcontainer.json [hunk 1]
```diff
@@ -13,5 +13,9 @@
 			]
 		}
 	},
-	"remoteUser": "simbricks"
-}
+	"remoteUser": "simbricks",
+	"remoteEnv": {
+		"PATH": "/home/simbricks/.local/bin:${containerEnv:PATH}"
+	},
+	"onCreateCommand": "pip install -r requirements.txt"
```
Comment (FreakyPenguin) [inline .devcontainer.json]: Why not just install these in the Dockerfile? Seems a bit silly to install locally for everyone opening the container if we can avoid it. Or am I missing something?

**Review by FreakyPenguin — CHANGES_REQUESTED** [review]
Other than the pip install this looks good to me.

File: .devcontainer.json (hunk 1 above)
Comment (jonas-kaufmann) [reply .devcontainer.json]: I agree. There's no particular reason not to do this in the Dockerfile directly.

**Comment by FreakyPenguin** [comment]
Could you also add the requirements.txt to the clean target variable at the bottom? So make clean deletes it and results in a clean tree.

**Review by FreakyPenguin — APPROVED** [review]
Looks good. Thanks for the changes!

**Pull request merged**